index.js

```diff
@@ -39,36 +146,38 @@ module.exports = {
               }
             }
         ],
-        "@typescript-eslint/no-unused-vars": ["error", {
-            "varsIgnorePattern": "^_",
-            "ignoreRestSiblings": true
-        }],
         "@typescript-eslint/explicit-function-return-type": ["error", {
             "allowExpressions": true
         }],
+        "@typescript-eslint/consistent-type-definitions": ["error", "type"],
```

Comment:
I think it's a good practice to use either types or interfaces (personally prefer types, but this rule can be changed or turned off if preferences differ or if the  project is a dependency of another that relies on a specific type definition style)